javascript/app-discover/src/components/lists/EdgeItem.tsx

```diff
@@ -20,10 +20,27 @@ export const EdgeItem: React.FC<EdgeItemProps> = memo(function EdgeItem({
 	columnName,
 	onFlip,
 	onRemove,
-	onPin,
+	onRemoveConstraint,
 	onSelect,
 	constraint,
+	flipAllowed,
 }) {
+	const isRejected = hasSameReason(ManualRelationshipReason.Removed, constraint)
+
+	const flipTooltip = useMemo((): string => {
+		return hasSameReason(ManualRelationshipReason.Flipped, constraint)
+			? 'Relationship manually reversed. Click to undo it'
```

Comment:
Let's avoid nested tertiary statements like this and just use some if-else blocks

Reply:
fixed, thanks!